app/scripts/modules/kubernetes/v2/manifest/wizard/manifestEntry.component.ts

```diff
@@ -0,0 +1,30 @@
+import { IComponentOptions, IController, module } from 'angular';
+
+import { IKubernetesManifestCommand } from '../manifestCommandBuilder.service';
+
+import './manifestEntry.less'
+
+class KubernetesManifestCtrl implements IController {
+  public command: IKubernetesManifestCommand;
+}
+
+class KubernetesManifestEntryComponent implements IComponentOptions {
+  public bindings: any = {command: '='};
+  public controller: any = KubernetesManifestCtrl;
+  public controllerAs = 'ctrl';
+  public template = `
+    <div class="container-fluid form-horizontal">
+      <ng-form name="manifest">
+        <div>
```

Comment:
How many divs are too many divs? (You might need this one, who knows).

Reply:
Is there such a thing as too many divs?